libraries/AP_DroneCAN/AP_DroneCAN.h

```diff
@@ -343,7 +346,11 @@ class AP_DroneCAN : public AP_CANDriver, public AP_ESC_Telem_Backend {
     Canard::ObjCallback<AP_DroneCAN, uavcan_protocol_debug_LogMessage> debug_cb{this, &AP_DroneCAN::handle_debug};
     Canard::Subscriber<uavcan_protocol_debug_LogMessage> debug_listener{debug_cb, _driver_index};
 
-#if AP_DRONECAN_VOLZ_FEEDBACK_ENABLED
+#if AP_DRONECAN_HIMARK_SERVO_SUPPORT && AP_SERVO_TELEM_ENABLED
+    Canard::ObjCallback<AP_DroneCAN, com_himark_servo_ServoInfo> himark_servo_ServoInfo_cb{this, &AP_DroneCAN::handle_himark_servoinfo};
```

Comment:
himark servo was just broken, there was a function but the callback was never registered.

Reply:
this must have been a regression, it worked when it went in

Reply:
In the change of DroneCAN library I think, I fixed the same thing for the volz message, https://github.com/ArduPilot/ardupilot/pull/27924 we should probably check all the rest....

Reply:
I have checked the rest, were OK no others missing.